src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferChoose.js

```diff
@@ -204,15 +204,15 @@ const KeyBasedAssetTransferChoose = ({
       }}
       footer={
         !isEmpty(keyBasedAssetsToTransfer) && (
-        <Footer>
-          <Button
-            title={t('button.next')}
-            onPress={() => {
-              calculateTransactionsGas(); // start calculating
-              navigation.navigate(KEY_BASED_ASSET_TRANSFER_CONFIRM);
-            }}
-          />
-        </Footer>
+          <FooterContent>
+            <Button
+              title={t('button.next')}
+              onPress={() => {
+                calculateTransactionsGas(); // start calculating
```

Comment:
When you say "start calculating", i'm assuming this is actually an "instant-ish" calculation (data payload * pre-fetched gas price from the SDK) or does it require some calculation? If the latter is true, should we not `await` this and then call navigate? 

I'm making some assumptions here about `calculateTransactionsGas()` 😂 but please feel free to clarify!

Reply:
This is redux action call, which is returns instantianously but there result will be reflected in redux state after a while.